improver_tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -785,10 +804,11 @@ def test_use_mask_cube_occurrences_masked_irregular(self):
                 [1.000000, 0.000000, 1.000000, 0.000000, 0.000000],
             ]
         )
```

Comment:
Don't think this statement has any effect - you already set up the cube with this data (as you tidied up, above).

Reply:
Need to clarify which statement you're referring to here, it looks like the expected array in  def test_use_mask_cube_occurrences_masked_irregular, but I don't think it is because that array is necessary for the expected data.

Reply:
It was about the masked data - but I misread it, sorry.  So please ignore the original comment.